funcx_endpoint/funcx_endpoint/endpoint/rabbit_mq/result_queue_subscriber.py

```diff
@@ -34,20 +43,21 @@ def __init__(
 
         """
         super().__init__()
-        self.conn_params = pika_conn_params
+        self.status = SubscriberProcessStatus.parent
+
+        self.conn_params = conn_params
         self.external_queue = external_queue
-        self.queue_name = "results"
-        self.routing_key = "*.results"
-        self.exchange_name = "results"
-        self.exchange_type = "topic"
+        self.kill_event = kill_event
+        self._channel_closed = multiprocessing.Event()
+        self._cleanup_complete = multiprocessing.Event()
+        self._watcher_poll_period = 0.1  # seconds
 
```

Comment:
Suggest putting the unit in the class var name directly.  Self-documenting.  Perhaps:
```python
self._watcher_poll_period_s = 0.1
```

Reply:
I did this and the log line change in an amend. I didn't have a meaningful commit message other than "fixup", so didn't think it was worth a separate commit.